### ISSUE TITLE
Getting Started 4x4 RGB Matrix Beaglebone

### ISSUE THREAD

**Issue by asadpiz** [Yona-Appletree/LEDscape#51]
Hi, I'm trying to control a [4x4 RGB LED Matrix WS2812](https://shop.mikroe.com/click/display/4x4-rgb) through beaglebone black and I'm able to run the opc server in demo mode i.e., all 16 LEDs fade with changing colour.

Now I'm finding it hard to go forward i.e., **to control individual LEDs:**
How does one control a single LED (the LED matrix that I'm using just uses 1 BB pin PWM/P9.14 and all LEDs are in a series so I assume it's just one strip).

 I am trying to use the [fadecandy python library](https://github.com/scanlime/fadecandy/blob/master/examples/python/opc.py) to connect to the opc server. Although the connectino is successful, the pixels that I send to the server are not showcased on the LEDs.

TL;DR: I'm trying to send pixels to the opc server but despite having connection, the pixels aren't put on the LEDs (all LEDs shutdown and I see one LED flicker once in a while).

Any example code of sending pixels to opc-server would be appreciated

**Comment by Yona-Appletree** [comment]
Have you tried the processing examples in this project? I'm afraid that if the LEDs turn on with the fade mode and you have a successful connection, helping you program them is out of scope for a bug slip.

However, the most likely issue is that LEDscape is configured to drive many separate channels, and you're not writing data into the right spot in the buffer.

Try running LEDscape with --strip-count 1, then figure out which pin to use, then get processing connected and see if it works. Good luck.
